modules/standard/FileSystem.chpl

```diff
@@ -50,7 +52,7 @@ proc chdir(out err: syserr, name: string) {
    Note: this is not safe within a parallel context.  A chdir call in one task
    will affect the current working directory of all tasks for that locale.
 */
-proc chdir(name: string) {
+proc locale.chdir(name: string) {
   var err: syserr = ENOERR;
   chdir(err, name);
```

Comment:
Oh, you're right, got it.  I was forgetting that this worked for secondary methods as well.  Duh.  Must be Friday.